ex8_ch3.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input: number of people eating pizza?
+pizza_eaters = int(input("How many people are eating pizza? "))
+
+#step2: get input: how many pizzas are there?
+total_pizzas = int(input("How many pizzas do you have? "))
+
+#step3: get input: how many slices are in a pizza?
+slices_per_pizza = int(input("How many slices are in a pizza? "))
+
+#step4: write calculation to see how many total slices there are
+total_pizza_slices = total_pizzas * slices_per_pizza
+
+#step5: write calculation to divide the pizza slices evenly among the total people
+slices_per_person = total_pizza_slices / pizza_eaters
+
+print(f"Each person can have {int(slices_per_person)} pieces of pizza.")
```

Comment:
Question: I had to add "int" here. If I don't, then I get a float. Why is that? Based on all of the variables above, this line of code should also be an int -- correct?

Reply:
It appears that in Python 3, the `/` operator results in true division, which means a `float` is always returned. There is a `int` division, the truncates the decimal: `//`.

I'm thinking the `//` would work here and should give you the same answer.

I actually had to look this up myself!

Here's a good article: http://www.informit.com/articles/article.aspx?p=1439189